### ISSUE TITLE
Parse angles from file pattern - window too large and can't advance

### ISSUE THREAD

**Issue by jmmuncie** [PreibischLab/BigStitcher#124]
Hi, I recently updated Bigstitcher to version 1.1.3 (was previously using 0.8.2) and have encountered a new error when attempting to define a new dataset. The automatic loader correctly identifies the patterns in my filenames (see first screenshot attached), but Bigstitcher then opens a window titled "Parse Angles from file pattern" that is too large to fully display and I am unable to advance beyond it (second screenshot).  

![Loader_01](https://user-images.githubusercontent.com/60235694/225385551-eed9082b-7f74-4539-9f3d-0b49c9563ca7.PNG)

![Parse_02](https://user-images.githubusercontent.com/60235694/225385683-c65d5252-47c8-4bc0-acb6-119437aad6cc.PNG)

**Comment by hoerldavid** [comment]
Hi @jmmuncie ,

Thanks for bringing this to our attention.
The bug was introduced by functionality I added but apparently did not test extensively - I deeply apologise for that.

Thankfully, [mhdominguez](https://github.com/mhdominguez) provided a pull request to the underlying multiview-reconstruction codebase fixing the issue (https://github.com/PreibischLab/multiview-reconstruction/pull/42). Thanks a lot!

I have merged the pull request and it should be fixed in the next update!

Best,
David
